src/index.js

```diff
@@ -69,14 +91,14 @@ export default function nodeResolve ( options = {} ) {
 						extensions: options.extensions
 					}, customResolveOptions ),
 					( err, resolved ) => {
+						if (options.browser && packageBrowserField) browserMapCache[resolved] = packageBrowserField;
+
 						if ( !disregardResult && !err ) {
 							if ( resolved && fs.existsSync( resolved ) ) {
 								resolved = fs.realpathSync( resolved );
 							}
 
-							if ( resolved === COMMONJS_BROWSER_EMPTY ) {
-								fulfil( ES6_BROWSER_EMPTY );
```

Comment:
We no longer need this line as we're not relying on `browser-resolve`s empty file.